src/14_legendre.jl

```diff
@@ -338,7 +333,7 @@ be casted to the type of `x`. Thus, variables `x,Am,Bm,Cm` are all of the same t
 > (p. 23)
 """
 function ABC_recurrence(n::Integer, x::T,
-        A::Union{Integer,Function}, B::Union{Integer,Function}, C::Union{Integer,Function},
+        ABC::Base.Generator,
```

Comment:
Generators aren't going to be better here, since they still aren't inlined…

Reply:
What about precomputing the coefficients `Am,Bm,Cm` and supplying them as arrays to the function `ABC_recurrence`?

Reply:
How big of an `n` would you precompute for?  For any practical size this will impose an arbitrary limit on the degree.

Reply:
removed that function in commit [780b6df](https://github.com/JuliaMath/SpecialFunctions.jl/pull/175/commits/780b6df504afa8169b6529f46b78dcd1b9350ef6).